rest-api/test/unit_test/api_test/participant_counts_over_time_api_test.py

```diff
@@ -0,0 +1,53 @@
+import datetime
+from clock import FakeClock
+
+from dao.participant_dao import ParticipantDao
+from model.hpo import HPO
+from dao.hpo_dao import HPODao
+from model.calendar import Calendar
+from dao.calendar_dao import CalendarDao
+from dao.participant_summary_dao import ParticipantSummaryDao
+from test.unit_test.unit_test_util import FlaskTestBase, random_ids
+from model.participant import Participant
+from participant_enums import OrganizationType, TEST_HPO_NAME, TEST_HPO_ID
+
+class ParticipantCountsOverTimeApiTest(FlaskTestBase):
+
+  def setUp(self):
+    super(ParticipantCountsOverTimeApiTest, self).setUp(use_mysql=True)
+    self.dao = ParticipantDao()
+    self.participant_summary_dao = ParticipantSummaryDao()
+    self.calendar_dao = CalendarDao()
+    hpo_dao = HPODao()
+    hpo_dao.insert(HPO(hpoId=TEST_HPO_ID, name=TEST_HPO_NAME, displayName='Test',
+                     organizationType=OrganizationType.UNSET))
+
+  def test_get_counts_with_default_parameters(self):
+    p = Participant()
+
+    time = datetime.datetime(2018, 1, 1)
+    with random_ids([1, 2]):
+      with FakeClock(time):
+        self.dao.insert(p)
+
+    curr_date = datetime.date(2017, 1, 1)
+    # Insert 1 years of dates
+    for _ in xrange(0, 365 * 1):
```

Comment:
Nit: you probably don't need 365 days to test your code. It will run a little faster if you create less.

You may also want this logic in setUp() so it will run for every test.

Reply:
Done.